BUILD

```diff
@@ -0,0 +1,7 @@
+python_requirements(
+    name="root",
+)
+
+python_test_utils(
+    name="test_utils0",
+)
```

Comment:
This is for `conftest.py`. I will rename the targets in this BUILD file (and others) in a follow-up PR.